### PR TITLE
Add warning for network path deprecation in Kodi native mode documentation

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
Considered removing the entire section but that would be confusing as the client still contains the native mode.

### CODE REVIEW

File: docs/general/clients/kodi.md [hunk 1]
```diff
@@ -95,6 +95,12 @@ Native mode accesses your media files directly from the filesystem, bypassing th
 
 To use Native mode, first set up your libraries in Jellyfin with a remote path.
 
+:::caution
+
+Starting from Jellyfin 10.9 it is no longer possible to set the shared network folder.
```
Comment (felix920506) [inline docs/general/clients/kodi.md:100]: Will existing setups stop working after upgrading the server to 10.9? If so then that should be documented.

Comment (nielsvanvelzen) [reply docs/general/clients/kodi.md:100]: nope, but we may remove the paths entirely in the future although nothing is decided.

**Review by felix920506 — APPROVED** [review]

**Review by Shadowghost — APPROVED** [review]

**Comment by nielsvanvelzen** [comment]
The future release label means "do not merge until the next release"

**Comment by Shadowghost** [comment]
Sorry, was on mobile and missed it.
Nonetheless in cases like this where the release is explicitly referenced I don't see a reason why we shouldn't merge it ahead of actual release.

**Comment by nielsvanvelzen** [comment]
Yeah for this specific PR it doesn't matter that much, but generally we want to wait with merging documentation for new versions until that version is available.

**Pull request merged**